src/Covid19Dashboard/index.jsx

```diff
@@ -43,6 +43,42 @@ class Covid19Dashboard extends React.Component {
     Object.entries(dashboardDataLocations).forEach(
       (e) => this.props.fetchDashboardData(e[0], e[1]),
     );
+
+    // scrolling effect added to show homepage starting from covid counts
+    const targetNode = document.getElementsByClassName('covid19-dashboard_panel')[0];
+
+    // defined a scroll flag to only scroll for one time
+    let scrollFlag = false;
+
+    // Options for the observer (which mutations to observe)
+    const config = { attributes: true, childList: true, subtree: true };
```

Comment:
does this observer needs to observe for attribute changes? looks like it is only looking for adding a node

Reply:
So according to [example](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver#example) here, We can use MutationObserver to observe changes in element either it changes the attribute or add a child list or a subtree. I have added condition to scroll the PRC homepage to show the Case Count div on top when map-slider div (which corresponds to the date slider at home page in PRC) is added in the HTML Document.

Reply:
right, this is what I meant, if you don't care about attribute changes, why seeting `attributes: true` in the observer config?

Reply:
Oh, that makes sense. Let me see if changing that works or not. Thank you so much for the note.

Reply:
Oh, that makes sense. I have changed setting `attributes: true` to `attributes: false` and made the commit